### PR TITLE
Implement get_network_instances()

### PR DESCRIPTION
**Pull request by bewing**
Code tested with vEOS 4.16.6M

### CODE REVIEW

**Comment by bewing** [comment]
Need to re-run tests after napalm-automation/napalm-base#139 is merged.

File: napalm_eos/eos.py [hunk 1]
```diff
@@ -1494,6 +1494,63 @@ def get_config(self, retrieve="all"):
...
+                u'route_distinguisher': u'',
+            },
+            u'interfaces': {
+                u'interface': {
+                    k: {} for k in all_interfaces if k not in all_vrf_interfaces.keys()
+                },
+            },
+        }
+
+        if name != '':
```
Comment (mirceaulinic) [inline napalm_eos/eos.py]: I would suggest:

``` python
if name:
    if name in vrfs:
        return {unicode(name): vrfs[name]}
   return {}
else:
    return vrfs
```

**Review by mirceaulinic — APPROVED** [review]
Just a minor change, otherwise looks good to me!

**Comment by mirceaulinic** [comment]
LGTM - let's see what @napalm-automation/council thinks

**Comment by dbarrosop** [comment]
+1

Merging

**Pull request merged**